service/minimal-config-example.toml

```diff
@@ -0,0 +1,43 @@
+# You will have to change *all* the values below to match your setup.
+#
+# Some of the config below are global graph network values, which you can find here:
+# https://github.com/graphprotocol/indexer/tree/main/docs/networks
+#
+# Pro tip: if you need to load some values from the environment into this config, you
+# can use shell expansions like with `envsubst`.
```

Comment:
By this comment I actually wanted to mean that it is supported and it is the same format as with envsubst. Perhaps not that clear though...
I wanted to add an example, but then of course it would try to expand it...

Reply:
Maybe a commented-out example

Reply:
> Maybe a commented-out example could help
That's the thing, the commented-out example will crash because it will try to expand it.
I don't want to go down the rabbit hole of handling TOML comments.

Reply:
Addressed in [5e629df](https://github.com/graphprotocol/indexer-rs/pull/160/commits/5e629dfb0964a757ae524bee3d1adf882fc1c911)